lua/params/control.lua

```diff
@@ -12,10 +12,11 @@ local tCONTROL = 3
 -- @param name of param
 -- @param controlspec
 -- @param formatter
-function Control.new(name, controlspec, formatter)
+function Control.new(id, name, controlspec, formatter)
```

Comment:
would it not make more sense to handle `id` outside of param classes - that is, in the paramset class instead, since ìd`s constitute keys of the paramset ?

Reply:
it makes sense, but i'd also like to keep the id in params so we can pass a param instance around and avoid looking up the id when we need it.